app/src/main/java/com/javahelp/frontend/MyViewModel.java

```diff
@@ -0,0 +1,48 @@
+package com.javahelp.frontend;
+
+import android.app.Application;
+import android.view.View;
+
+import androidx.annotation.NonNull;
+import androidx.lifecycle.AndroidViewModel;
+import androidx.lifecycle.MutableLiveData;
+import androidx.lifecycle.ViewModel;
+
+public class MyViewModel extends AndroidViewModel {
+    private MutableLiveData<String> userName;
+    private MutableLiveData<String> passWord;
+
+    public MyViewModel(@NonNull Application application) {
```

Comment:
I'm thinking of a case where a constructor takes in `null` application and handle that exception. Is it necessary?

Reply:
Will have a look of that